src/AppRouter.js

```diff
@@ -215,10 +219,16 @@ const AppRouter = () => (
                     <CreateOfferPage />
                 </PageLayout>
             </Route>
-            <Route
-                path="/error"
-                key="/error"
-            >
+            <Route exact path="/whatsNew">
```

Comment:
This could be whats-new, to be consistent with privacy-policy and terms-and-conditions